modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -2237,7 +2237,7 @@ private void ackNodeMetrics(DecimalFormat dblFmt,
 
             msg.nl()
                 .a("Metrics for local node (to disable set 'metricsLogFrequency' to 0)").nl()
-                .a("    ^-- Node [id=").a(id).a(name() != null ? ", name=" + name() : "").a(", uptime=")
+                .a("    ^-- Node [id=").a(id).a(", consistentId=").a(localNode().consistentId()).a(name() != null ? ", name=" + name() : "").a(", uptime=")
```

Comment:
It seems to me, you can use the `locNode` variable that is already defined in the scope of this method instead of using `localNode()`